age_to_bucket_parser.py

```diff
@@ -0,0 +1,18 @@
+import json
+import yaml
+with open('hw.json') as hw_json_file:
+    json_data = json.load(open('hw.json'))
+ppl_age = json_data['ppl_ages']
+age_buckets = json_data['buckets']
+age_limits = [max(ppl_age.values())+1]
+age_buckets.extend(age_limits)
```

Comment:
You're missing the names between 0 to 11:
`- 0-11: [Dana, Danail]`

For example - adding 
`age_buckets.extend([0])` will do the trick

Try think of other ways of achieving this